### PR TITLE
docs(aurora): fix permission scopes

### PR DESCRIPTION
**Pull request by 317brian**

### CODE REVIEW

File: content/en/docs/installation/aurora-install.md [hunk 1]
```diff
@@ -64,22 +64,26 @@ Register your Armory Enterprise environment so that it can communicate with Armo
 
 ## Create client credentials for your Agents
 
+Use client credentials for authentication and authorization for Armory's hosted cloud services. For example, RNAs use these credentials to authenticate itself to Armory's hosted cloud services.
```
Comment (aimeeu) [inline content/en/docs/installation/aurora-install.md:67]:
```suggestion
Use client credentials for authentication and authorization for Armory's hosted cloud services. Services such as RNAs use these credentials to authenticate to Armory's hosted cloud services.
```

File: content/en/docs/installation/aurora-install.md [hunk 2]
```diff
@@ -64,22 +64,26 @@ Register your Armory Enterprise environment so that it can communicate with Armo
...
 
    {{< figure src="/images/deploy-engine/cloud-env-context.png" alt="The upper right section of the window shows what environment you are currently in." >}}
 
-1. In the left navigation menu, select **Access Management > Client Credentials**.
-2. In the upper right corner, select **New Credential**.
-3. Create a credential for your RNAs. Use a descriptive name for the credential, such as `Armory K8s Agent`
-4. Set the permission scope to the following:
+3. In the left navigation menu, select **Access Management > Client Credentials**.
+4. In the upper right corner, select **New Credential**.
+5. Create a credential for your RNAs. Use a descriptive name for the credential, such as `RNA for TV App K8s`
```
Comment (aimeeu) [inline content/en/docs/installation/aurora-install.md:76]:
```suggestion
5. Create a credential for your RNA. Use a descriptive name for the credential, such as `RNA for TV App K8s`
```

**Review by aimeeu — CHANGES_REQUESTED** [review]
few nits

**Comment by 317brian** [comment]
I'm going to close this since CDF is going to fix the pre-configured scope in the UI.

**Pull request closed without merging**